### PR TITLE
fbanados in coq.

### PR DESCRIPTION
**Pull request by fbanados**
A coq verified implementation.

### CODE REVIEW

File: people.json [hunk 1]
```diff
@@ -187,5 +187,16 @@
...
+        "github": "fbanados",
+        "name": "Felipe Bañados Schwerter",
+        "title": "PhD Candidate, UBC",
+        "entries": [
+            {
+                "name": "coq",
+                "link": "./entries/fbanados/fib.v"
+            }
+        ]
+    },
```
Comment (braxtonhall) [inline people.json]: Sadly, trailing comma will break the website :(

**Review by braxtonhall — CHANGES_REQUESTED** [review]
JSON is dumb

File: people.json (hunk 1 above)
Comment (fbanados) [reply people.json]: ok!

**Comment by fbanados** [comment]
Done!

**Review by braxtonhall — APPROVED** [review]
🫰

**Pull request merged**